Server/Plugins/APIDump/Classes/World.lua

```diff
@@ -3032,7 +3032,18 @@ function OnAllChunksAvailable()</pre> All return values from the callbacks are i
 						Type = "number",
 					},
 				},
-				Notes = "Sets the blockticking to start at the specified block in the next tick.",
+				Notes = "DEPRECATED, use SetNextBlockToTick() instead.",
```

Comment:
Just remove it already...

Reply:
It is used in plugins, it gets a Deprecation.